sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py

```diff
@@ -40,7 +40,8 @@ class CertificateClient(KeyVaultClientBase):
     :param str vault_url: URL of the vault the client will access. This is also called the vault's "DNS Name".
     :param credential: An object which can provide an access token for the vault, such as a credential from
         :mod:`azure.identity`
-    :keyword str api_version: version of the Key Vault API to use. Defaults to the most recent.
+    :keyword api_version: version of the Key Vault API to use. Defaults to the most recent.
+    :paramtype api_version: ~azure.keyvault.certificates.ApiVersion
```

Comment:
Our other enums allow arbitrary string values because they subclass `str`; that isn't a feature of `Enum`. `ApiVersion` subclasses only `Enum` because it doesn't need to be forward compatible.

However, it's true a string works for `api_version` here, provided that string is a valid `ApiVersion` value. That's by design, to prevent a breaking change for anyone passing a string. I mention only `ApiVersion` in the docstring to point everyone else at the type they should be using.